Adyen/Assets/Generated/LocalizationKey.swift

```diff
@@ -2,7 +2,7 @@
 // Copyright (c) 2021 Adyen N.V.
 //
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
-// This file is autogenerated. Please do not modify it.
```

Comment:
It is still auto-generated, right? Why remove header?

Reply:
I haven't touched this file, I just updated translations and build with Xcode.

Reply:
I think the swift format and the translations script have conflict, lets fix this in a another PR